cli/cmd/messaging.go

```diff
@@ -0,0 +1,56 @@
+package cmd
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/authorhealth/go-elation"
+	"github.com/davecgh/go-spew/spew"
+	"github.com/spf13/cobra"
+)
+
+var (
+	findThreadMembersPatients []int64
+	findThreadMembersUsers    []int64
+)
+
+var findThreadMembersCmd = &cobra.Command{
+	Use: "find-thread-members",
+	Run: wrapRunFunc(func(ctx context.Context, client *elation.Client, args []string) error {
+		response, _, err := client.ThreadMemberSvc.Find(ctx, &elation.FindThreadMembersOptions{
+			Patient: findThreadMembersPatients,
+			User:    findThreadMembersUsers,
+		})
+		if err != nil {
+			return err
+		}
+
+		spew.Dump(response)
```

Comment:
spew.Dump call

Reply:
That's on purpose. I don't want to write formatters for every response type.